### PR TITLE
637 variable assignment

### PR DESCRIPTION
**Pull request by dwang429**
### What are the issues this pull addresses (issue numbers / links)?
637

### Did you add tests to cover your changes (yes/no)?
no

### Did you update the documentation accordingly (yes/no)?
yes

### Did you read the CONTRIBUTING document (yes/no)?
yes

### Summary
This PR allows for the proper passing of variables to newly opened Ansys designs. It is built upon renderer-analysis-split and should only be merged after it.
Screen Shot 2021-07-14 at 1 36 44 PM
Screen Shot 2021-07-14 at 1 36 52 PM
Screen Shot 2021-07-14 at 1 37 12 PM

### Details and comments
Before, variables could be introduced to new Ansys designs but they weren't assigned correctly. For example, variables Lj1 and Cj1 could represent a junction inductance and capacitance, respectively, and they'd show up in a newly created Ansys design, but when you examine a junction's boundary condition (a junction whose inductance/capacitance have been assigned to said variables) by double clicking on it, it didn't take on the proper values. Now it does. (See screenshots above)

### CODE REVIEW

File: qiskit_metal/renderers/renderer_ansys/ansys_renderer.py
Comment (marcolincs) [inline qiskit_metal/renderers/renderer_ansys/ansys_renderer.py:1533]: As we wait for the analysis split merge, how about we add here a "else" to logger.warn people that they need to create a design before they try to set variables, else their rendering will pick up value 0 for all the variables?

File: qiskit_metal/analyses/quantization/sim_eigenmode.py
Comment (marcolincs) [inline qiskit_metal/analyses/quantization/sim_eigenmode.py:130]: this will need to change to `self.sim.setup.vars` after #642 is merged. Unless we merge this one first, then we need to change it inside #642 before merging.

Comment (dwang429) [reply qiskit_metal/analyses/quantization/sim_eigenmode.py:130]: Do you mean `self.setup.vars`?

Comment (marcolincs) [reply qiskit_metal/analyses/quantization/sim_eigenmode.py:130]: yes

File: qiskit_metal/renderers/renderer_ansys/ansys_renderer.py
Comment (dwang429) [reply qiskit_metal/renderers/renderer_ansys/ansys_renderer.py:1533]: Done.

Comment (marcolincs) [reply qiskit_metal/renderers/renderer_ansys/ansys_renderer.py:1537]: thank you!

File: qiskit_metal/analyses/quantization/sim_eigenmode.py
Comment (marcolincs) [reply qiskit_metal/analyses/quantization/sim_eigenmode.py]: looks good, also the other catches of the same around the code.

**Review by marcolincs — APPROVED** [review]

**Review by JeremyDrysdale-ibm — APPROVED** [review]

**Review by zlatko-minev — APPROVED** [review]

**Review by ThomasGM4 — APPROVED** [review]

**Pull request merged**
